### ISSUE TITLE
Increase means for mappers to communicate with each other, if needed. 

### ISSUE THREAD

**Issue by pgiraud** [hotosm/osm-tasking-manager#40]
Could be simply a dedicated IRC room, or a chat window in the App per Job

(reported by Mikel Maron)

**Comment by Augustind** [comment]
I totally agree about this need, thinking about it this morning, working on the job on Goré, Chad. Tx!
If this place is dedicated only for solution proposal, just tell me, I'm not sure.
